jsonrpc/eth_test.go

```diff
@@ -401,3 +401,40 @@ func TestEthFeeHistory(t *testing.T) {
 	assert.NoError(t, err)
 	assert.NotNil(t, fee)
 }
+
+func TestEthMaxPriorityFeePerGas(t *testing.T) {
+	s := testutil.DeployTestServer(t, nil)
```

Comment:
Can you use `s := testutil.NewTestServer(t)` instead? I am trying not to deploy a new server for each unit test as it was done before. Otherwise, it taxes too much the CI.

Reply:
Fixed https://github.com/umbracle/ethgo/pull/251/commits/2be27397e624f23ce1a56c6c3104630adfe6f2e4